changelog.txt

```diff
@@ -8,6 +8,42 @@ Legend
 + : added
 - : removed
 
+2022-12-14 v1.120.0
+-------------------
+
++ Add "Sponsor us" to page footer (#5882) 
++ Git Submodules support (#5883) 
```

Comment:
I think this does not add support as such? It just fixes it so there is no error